src/ser/tests.rs

```diff
@@ -121,6 +121,16 @@ fn test_byte_stream() {
         "(0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15)"
     );
 
+    // Requires padding of the base64 data
+    let large = vec![0x01, 0x02, 0x03, 0x04];
+    let large = serde_bytes::Bytes::new(&large);
+    assert_eq!(to_string(&large).unwrap(), "\"AQIDBA==\"");
+
+    // Requires no padding of the base64 data
+    let large = vec![0x01, 0x02, 0x03, 0x04, 0x05, 0x06];
+    let large = serde_bytes::Bytes::new(&large);
+    assert_eq!(to_string(&large).unwrap(), "\"AQIDBAUG\"");
+
     let large = vec![255u8; 64];
     let large = serde_bytes::Bytes::new(&large);
     assert_eq!(
```

Comment:
The one just below is already would fail if the URL-safe version instead of the standard version is used.

Let me know if you have any other strings / kind of strings you'd like to have tested. Padding handling would be covered with the new tests now, and they pass before/after.

Reply:
Those look good - could you add one for the bytes error case as well where we use base64 to format the erroneously given bytes?

Reply:
I'm not entirely sure if you meant this, but what I just added seems like a good addition too.

Reply:
Absolutely, the ones you have already added are good. I am referring to https://github.com/ron-rs/ron/blob/0b7446179df381eaa215eb0eaa67f2ba68d1bca6/src/error.rs#L305 - could this encode also get a similar test?

Reply:
Ah! Sure :)

Reply:
Added something. I tried a lot to trigger this specific error, and that's the easiest I could find. If you have a better idea let me know.